server.go

```diff
@@ -138,9 +166,8 @@ func (svr *Server) sftpServerWorker(pktChan chan orderedRequest) error {
 		// If server is operating read-only and a write operation is requested,
 		// return permission denied
 		if !readonly && svr.readOnly {
-			svr.sendPacket(orderedResponse{
-				responsePacket: statusFromError(pkt, syscall.EPERM),
-				orderid:        pkt.orderID()})
+			svr.pktMgr.readyPacket(
+				svr.pktMgr.newOrderedResponse(statusFromError(pkt, syscall.EPERM), pkt.orderID()))
```

Comment:
I know the original code ends the last line with a `)`, but if one is going to break things up onto separate lines, each argument should go on its own line, ending with a `,`, and then the ending `)` should be on its own line, the same as one should fill in the fields of a struct with `{` and `}`.